sentry-android-core/src/main/java/io/sentry/android/core/PerformanceAndroidEventProcessor.java

```diff
@@ -41,6 +57,20 @@
       }
     }
 
+    final SentryId eventId = transaction.getEventId();
+    final SpanContext spanContext = transaction.getContexts().getTrace();
+
+    // only add slow/frozen frames to transactions created by ActivityLifecycleIntegration
```

Comment:
We're technically adding to transactions named `UI_LOAD_OP` which could set by the user.

```suggestion
    // only add slow/frozen frames to transactions created by ActivityLifecycleIntegration
    // which have the operation UI_LOAD_OP. If a user-defined (or hybrid SDK) transaction 
    // users it, we'll also add the metrics if available
```

Worth clarifying it?